R/backend-.R

```diff
@@ -282,9 +282,13 @@ base_scalar <- sql_translator(
   str_sub = sql_str_sub("SUBSTR"),
   str_like = function(string, pattern, ignore_case = TRUE) {
     if (isTRUE(ignore_case)) {
```

Comment:
If you can find a authoritative link that states that LIKE is case sensitive, it would be great to include it here so we don't make the same mistake again in the future.

Reply:
I have added a link to https://docs.getdbt.com/sql-reference/like (not sure though if this is considered an authoritative link or not, but hopefully somewhere close)